repositories/src/main/scala/cromwell/pipeline/datastorage/dto/Run.scala

```diff
@@ -0,0 +1,34 @@
+package cromwell.pipeline.datastorage.dto
+
+import java.sql.Timestamp
+
+import cromwell.pipeline.model.wrapper.{RunId, UserId}
+import play.api.libs.json.{Format, JsResult, JsValue, Json, OFormat}
+
+final case class Run(
+  runId: RunId,
+  projectId: ProjectId,
+  projectVersion: String,
+  status: String,
```

Comment:
Use ADT instead of `String`. For example, there's a `Visibility` trait that solves the same problem.
@ilyaonishenko Maybe it's time to use some suitable library, like `enumeratum`?

Reply:
I don't get it, why use Visibility? Is there something that we need to wrap it?

Reply:
You shouldn't use `Visibility` at all. It's just an example of how to introduce ADT and integrate it with Slick.

Reply:
I created a new dto called ```CustomsWithEnumSupport```. In this file i placed the ```Visibility``` and ```Status```